source/clear-linux/tutorials/multi-boot/multi-boot-restore-bl.rst

```diff
@@ -14,43 +15,48 @@ longer the default boot loader. To restore it, follow the steps below.
 
    .. figure:: figures/multi-boot-restore-bl-1.png
 
-      Figure 1: |CL|: Console
+      Figure 1: |CL|: Console.
 
```

Comment:
@bktan8, @tpleavitt : Since all commands in this guide are done as root, I left the step below as-is, based on my notes from the WW17 meeting. OK?